e2e/framework/manager/rook_test_infra.go

```diff
@@ -364,6 +376,20 @@ func (r *rookTestInfraManager) InstallRook(tag string) (err error, client contra
 	if err != nil {
 		panic(err)
 	}
+	err = r.copyImageToNode(r.getContainerIdByName(k8s_master_container_name), rookClientTag)
+	if err != nil {
+		panic(err)
+	}
+
+	err = r.copyImageToNode(r.getContainerIdByName(k8s_node1_container_name), rookClientTag)
```

Comment:
looks like this already existed, but the naming convention for `k8s_node1_container_name` would be more like `node1ContainerName`